### PR TITLE
Measurement search criteria

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://github.com/ibexa/measurement/pull/38
| Versions      | 4.2, master

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Code samples are working
- [x] PHP code samples have been fixed with PHP CS fixer
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/guide/search/criteria_reference/rangemeasurementattributemaximum_criterion.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# RangeMeasurementAttributeMaximum Criterion
+
+The `RangeMeasurementAttributeMaximum` Search Criterion searches for products by the maximum value of their measurement (range) attribute.
+
+## Arguments
+
+-  `identifier` - string representing the attribute.
+-  `value` - `SimpleValue` object representing the maximum attribute value.
```
Comment (adamwojs) [inline docs/guide/search/criteria_reference/rangemeasurementattributemaximum_criterion.md:8]:
```suggestion
-  `value` - `\Ibexa\Contracts\Measurement\Value\SimpleValueInterface` object representing the maximum attribute value.
```

File: docs/guide/search/criteria_reference/rangemeasurementattributeminimum_criterion.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# RangeMeasurementAttributeMinimum Criterion
+
+The `RangeMeasurementAttributeMinimum` Search Criterion searches for products by the minimum value of their measurement (range) attribute.
+
+## Arguments
+
+-  `identifier` - string representing the attribute.
+-  `value` - `SimpleValue` object representing the minimum attribute value.
```
Comment (DominikaK) [inline docs/guide/search/criteria_reference/rangemeasurementattributeminimum_criterion.md:8]:
```suggestion
- `value` - `\Ibexa\Contracts\Measurement\Value\SimpleValueInterface` object representing the minimum attribute value.
```

File: docs/guide/search/criteria_reference/simplemeasurementattribute_criterion.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# SimpleMeasurementAttribute Criterion
+
+The `SimpleMeasurementAttribute` Search Criterion searches for products by the minimum of their measurement (single) attribute.
+
+## Arguments
+
+-  `identifier` - string representing the attribute.
+-  `value` - `SimpleValue` object representing the attribute value.
```
Comment (DominikaK) [inline docs/guide/search/criteria_reference/simplemeasurementattribute_criterion.md:8]:
```suggestion
-  `value` - `\Ibexa\Contracts\Measurement\Value\SimpleValueInterface` object representing the attribute value.
```

**Review by adamwojs — APPROVED** [review]

**Review by dabrt — APPROVED** [review]

File: docs/guide/search/criteria_reference/simplemeasurementattribute_criterion.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# SimpleMeasurementAttribute Criterion
+
+The `SimpleMeasurementAttribute` Search Criterion searches for products by the minimum of their measurement (single) attribute.
```
Comment (dabrt) [inline docs/guide/search/criteria_reference/simplemeasurementattribute_criterion.md:3]: Is it the minimum value or rather the exact value of the measurement attribute?

Comment (DominikaK) [inline docs/guide/search/criteria_reference/simplemeasurementattribute_criterion.md:3]:
```suggestion
The `SimpleMeasurementAttribute` Search Criterion searches for products by the value of their measurement (single) attribute.
```

**Pull request merged**